definitions/infra-awsapigatewayresource/definition.yml

```diff
@@ -0,0 +1,3 @@
+domain: INFRA
+type: AWSAPIGATEWAYRESOURCE
+goldenTags: [aws.accountId, aws.region, aws.availabilityZone, aws.Project, aws.Environment, aws.CostCenter]
```

Comment:
nit: can you provide this in the format:

```
goldenTags:
- tag
- secondTAg
```

Will be easier to avoid conflicts in the future this way